api-samples/downloads/downloads_overwrite/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "Downloads Overwrite Existing Files",
+  "description": "All downloads overwrite existing files instead of adding ' (1)', ' (2)', etc.",
```

Comment:
```suggestion
  "description": "Uses the [`chrome.downloads.onDeterminingFilename`](https://developer.chrome.com/docs/extensions/reference/downloads/#event-onDeterminingFilename) handler to override target file names for all downloads instead of adding ' (1)', ' (2)', etc.",
```